src/app/features/vendor/models/vendor.model.ts

```diff
@@ -9,4 +9,27 @@ export interface IVendor {
   phone: string;
   website?: string;
   isApproved: boolean;
+}
+
+export interface IVendorCreation {
+  name: string;
+  city: string;
+  stateProvinceRegion: string;
+  country: string;
+  email: string;
+  phone: string;
+  website: string;
+  serviceId: number;
+  isApproved: boolean;
+  marketIds: number[];
+}
+
+export interface IService {
+  id: number;
+  name: string;
+}
+
+export interface IMarket {
```

Comment:
duplicate interface, rename the interface use it for all drop down data

Reply:
done